matlab_simulation/05-estimation/kalman_filter/measurement_model.m

```diff
@@ -0,0 +1,53 @@
+function [C,D,Q,m] = measurement_model(example)
+
+% Provides discretized C(measurement), D(feed-forward) and Q(disturbance) 
+% matrices for the measurement model
+
+% Inputs:
+%       example   : Example under considertaion [1,2 or 3]
+
+% Outputs:
+%       C         : Measurement
+%       D         : Feed-forward
+%       Q         : Disturbance
+%       m         : Measurement dimension
+
+% Example 1: Temperature
+if example == 1
+    C = 1;
+    D = 0;
+    Q.Q = 4;
+    Q.QR = 4;
```

Comment:
I am not quite sure why there is Q.Q and Q.QR. In Kalman I you used Q.Q and in Kalman II you used Q.QR. Are they there to show the effect on Kalman gain when the disturbance changes?